src/Model/Errors/ErrorList.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Model\Errors;
+
+class ErrorList
```

Comment:
Zmień może na ErrorsList albo ErrorsContainer

Reply:
Dodałbym też konstruktor w którym był przyjął int $httpStatusCode = Response::BAD_REQUEST + getter i setter i wtedy jak zwracasz response z containera to zamiast ręcznie wpisywać 400 to zrób $errorsContainer->getStatusCode();